bftengine/include/bftengine/IRequestHandler.hpp

```diff
@@ -55,7 +56,9 @@ class IRequestsHandler {
   std::shared_ptr<concord::reconfiguration::IReconfigurationHandler> getReconfigurationHandler() const {
     return reconfig_handler_;
   }
-  virtual void setReconfigurationHandler(std::shared_ptr<concord::reconfiguration::IReconfigurationHandler> rh) {
+  virtual void setReconfigurationHandler(std::shared_ptr<concord::reconfiguration::IReconfigurationHandler> rh,
+                                         concord::reconfiguration::ReconfigurationHandlerType type =
+                                             static_cast<concord::reconfiguration::ReconfigurationHandlerType>(1)) {
```

Comment:
Should we use REGULAR here for better readability

Reply:
We can't. Here we use forward declaration of the enum.